packages/database-version-control/src/migrations/2023.01.04T15.26.22.create-application-reference-table.sql

```diff
@@ -1,6 +1,6 @@
 CREATE TABLE IF NOT EXISTS bng.application_reference
 (
 	application_reference_id UUID CONSTRAINT pk_application_reference_application_reference_id PRIMARY KEY DEFAULT uuid_generate_v4(),
-	application_reference VARCHAR(20) CONSTRAINT uq_application_reference_application_reference UNIQUE DEFAULT null,
+	application_reference VARCHAR(22) CONSTRAINT uq_application_reference_application_reference UNIQUE DEFAULT null,
```

Comment:
This needs a new migration to alter the column length as the migration has been merged to master and deployed to cloud environments.